client/css/style.css

```diff
@@ -1365,11 +1365,27 @@ background on hover (unless active) */
 }
 
 #chat .error,
-#chat .error .from,
-#chat .channel .highlight .from,
-#chat .channel .highlight .text,
-#chat .channel .highlight .user {
-	color: #f00;
+#chat .error .from {
+	color: #e74c3c;
+}
+
+#chat .message.highlight {
```

Comment:
I realized that I accidentally removed `.channel` and therefore this highlights in queries too. I was about to re-add it, then wondered if `.highlight` class should be in queries at all. Any insight, @xPaw?